client/commonFramework/update-preview.js

```diff
@@ -13,7 +13,10 @@ window.common = (function(global) {
   window.__err = null;
   window.loopProtect.hit = function(line) {
     window.__err = new Error(
-      'Potential infinite loop at line ' + line
+      'Potential infinite loop at line ' + line '. You can '  
+       + 'disable loop protection by adding "//noprotect" to your code; ho'
+       + 'wever, be aware that by disabling this feature, you run the risk ' 
+       + 'of your browser crashing should an infinite loop occur.';
```

Comment:
Can't have a semi-colon here:
`+ 'of your browser crashing should an infinite loop occur.';` 
This is a function call, not a variable assignment like the above.
(I made exactly the same error when testing a fix locally ;) )